src/sentry/grouping/api.py

```diff
@@ -162,30 +223,28 @@ def get_default_enhancements(config_id=None) -> str:
     return Enhancements(rules=[], bases=[base]).dumps()
 
 
-def get_projects_default_fingerprinting_bases(
-    project: Project, config_id: str | None = None
-) -> Sequence[str] | None:
-    """Returns the default built-in fingerprinting bases (i.e. sets of rules) for a project."""
-    from sentry.projectoptions.defaults import DEFAULT_GROUPING_CONFIG
+def get_default_fingerprinting(config_id: str | None = None) -> dict[Any, Any]:
+    """Returns the default fingerprinting rules."""
+    bases: Sequence[str] | None = DEFAULT_GROUPING_FINGERPRINTING_BASES
 
-    config_id = (
-        config_id
-        # TODO: add fingerprinting config to GroupingConfigLoader and use that here
-        or PrimaryGroupingConfigLoader()._get_config_id(project)
-        or DEFAULT_GROUPING_CONFIG
-    )
+    if config_id is not None:
+        bases = CONFIGURATIONS[config_id].fingerprinting_bases
 
-    bases = CONFIGURATIONS[config_id].fingerprinting_bases
-    return bases
+    # TODO: include_builtin ought to depend of a feature flag, but this is generic function that doesn't depend on project
+    return FingerprintingRules([], bases=bases).to_json(include_builtin=False)
 
 
-def get_default_grouping_config_dict(id=None) -> GroupingConfig:
+def get_default_grouping_config_dict(config_id: str | None = None) -> GroupingConfig:
```

Comment:
If you could handle typing changes in another PR, it would make reviewing this PR a little easier.